blog/2023-10-1-cloudgaming/index.md

```diff
@@ -0,0 +1,88 @@
+---
+slug: Cloud-gaming
+title: Exploring the Future of Gaming: Cloud Gaming Unveiled
```

Comment:
In Docusaurus, the contents written within the dashes `(---)` are in YAML format. Fundamentally, a file written in YAML consists of a set of **key-value pairs**. Each pair is written as `key : value`. Spaces are optional.

In your title you have a colon `:` which highlights incomplete explicit mapping pair, which means either a key node is missing or a value isn't there. There errors are tough to spot in the live server or local build, but are sure to break production, exactly like in our case.

```suggestion
title: Exploring the Future of Gaming - Cloud Gaming Unveiled
```

Reply:
Ohhhkkk